drivers/display/ssd1327_regs.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 Savoir-faire Linux
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SSD1327_REGS_H__
+#define __SSD1327_REGS_H__
+
+/*
+ * Fundamental Command Table
+ */
+#define SSD1327_SET_COLUMN_ADDR			0x15
+#define SSD1327_SET_ROW_ADDR			0x75
+
+#define SSD1327_SET_CONTRAST_CTRL		0x81
+
+#define SSD1327_SET_SEGMENT_MAP_REMAPED		0xa0
+#define SSD1327_SET_DISPLAY_START_LINE		0xa1
+#define SSD1327_SET_DISPLAY_OFFSET		0xa2
+
+#define SSD1327_SET_NORMAL_DISPLAY		0xa4
+#define SSD1327_SET_ENTIRE_DISPLAY_ON		0xa5
+#define SSD1327_SET_ENTIRE_DISPLAY_OFF		0xa4
```

Comment:
Is this intentional it is the same value as the display_normal constant?

Reply:
Good catch, the value for SSD1327_SET_ENTIRE_DISPLAY_OFF is 0xa6 following the datasheet